Refactor Spec2Vec and allow both Spectrum and SpectrumDocument input (#54)
iomega/spec2vec: CHANGELOG.md

## [Unreleased]

## Changed

> - refactored `Spec2Vec` to now accept both `Spectrum` and `SpectrumDocument` input [#51](https://github.com/iomega/spec2vec/issues/51)

Reviewer: Sounds like you need to give Spec2Vec both Spectrum and SpectrumDocument at the same time while either one is required.
```suggestion
- refactored `Spec2Vec` to now accept `Spectrum` or `SpectrumDocument` as input [#51](https://github.com/iomega/spec2vec/issues/51)
```
